packages/Frontend/config/webpack.config.dev.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
       inject: false,
       links: ['https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.12/semantic.min.css'],
       template,
+      mobile: true,
```

Comment:
This should probably go into it's own pull request. If I request other changes and they're going to take a while to do you can separate this out into it's own pull request and I'll approve, it's fine to leave in though, it just doesn't really relate directly the small card.